src/PostRector/Rector/ClassRenamingPostRector.php

```diff
@@ -88,19 +88,20 @@ public function afterTraverse(array $nodes): array
         return $nodes;
     }
 
-    /**
-     * @param array<string, string> $oldToNewClasses
-     */
-    private function resolveResultWithPhpAttributeName(
-        Name $name,
-        array $oldToNewClasses,
-        ?Scope $scope
-    ): ?FullyQualified {
+    public function shouldTraverse(array $stmts): bool
+    {
+        $this->oldToNewClasses = $this->renamedClassesDataCollector->getOldToNewClasses();
```

Comment:
We calculate the old to new classes here so that they don't need to be calculated for every node